pkgs/development/tools/build-managers/apache-ant/default.nix

```diff
@@ -78,6 +78,10 @@ stdenv.mkDerivation rec {
       chmod +x $out/bin/ant
     ''; # */
 
+  passthru = {
+    updateScript = ./update.sh;
```

Comment:
Does `gitUpdater` work for this?

Reply:
Indeed; I forgot about `gitUpdater` :sweat_smile: Used in latest push.